changelog/unreleased/pull-4998

```diff
@@ -0,0 +1,8 @@
+Bugfix: Fix extended attributes handling for VSS snapshots
```

Comment:
Please extend the changelog entry from https://github.com/restic/restic/pull/4980 . From an end user perspective the changelog describes what has changed between two released restic versions.

Reply:
done